ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -804,9 +840,14 @@ then
 	# Try to get an USB console
 	nshterm /dev/ttyACM0 &
 else
-	mavlink start -r 800000 -d /dev/ttyACM0 -m config -x
+	# Don't start mavlink in passthrough mode because we currently can't seem to stop it.
```

Comment:
It will make upgrading impossible (other than just on plugging it in). This is not something we can merge.